schemas/aws/cleanup-option-1.yml

```diff
@@ -39,5 +42,19 @@ oneOf:
   - provider
   - regex
   - age
+- properties:
+    provider:
+      type: string
+      description: Set CloudWatch log retention
+      enum:
+      - cloudwatch
+    regex:
+      type: string
+    retention_in_days:
+      type: string
```

Comment:
Since `retention_in_days` already indicates the measuring unit. I suggest removing the suffix 'd' and changing the type to integer. Besides, we could set an enum for the values, for reference https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html#API_PutRetentionPolicy_RequestSyntax